src/libraries/System.Threading/tests/ThreadLocalTests.cs

```diff
@@ -369,6 +369,7 @@ public static void RunThreadLocalTest9_Uninitialized()
 
         [Fact]
         [OuterLoop]
+        [PlatformSpecific(~TestPlatforms.Browser)] // Operation is not supported on this platform.
```

Comment:
Just curious, what operation is not supported here?

Reply:
[FAIL] System.Threading.Tests.ThreadLocalTests.ValuesGetterDoesNotThrowUnexpectedExceptionWhenDisposed
  info: System.PlatformNotSupportedException : Operation is not supported on this platform.
  info:    at System.Threading.Thread.Start()
  info:    at System.Threading.Tests.ThreadLocalTests.ValuesGetterDoesNotThrowUnexpectedExceptionWhenDisposed()
  info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo cultu

Reply:
should be `[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]` then though?